release/models/system/openconfig-system-management.yang

```diff
@@ -127,6 +127,77 @@ module openconfig-system-management {
     }
   }
 
+
+  grouping system-netconf-config {
+    description
+      "Configuration data for the netconf ptotocol";
+
+    leaf ssh {
+      type boolean;
+      default true;
+      description
+        "Enables the netconf protocol. The netconf is enabled by
```

Comment:
Should this read something more like....
"Enables the NETCONF server, using the SSHv2 transport. The NETCONF server is enabled by default."

Reply:
Services enabled by default is questionable practice but the NETCONF subsystem is also dependent off the `protocol-version` being correctly set in `/system/ssh-server`

Consistency with other services would have a separate `enable` leaf as well

Reply:
comments incorporated . pls review

Reply:
There is still no `enable` leaf and your description under the transport protocol implies the NETCONF subsystem is enabled by default